sigle/src/ui/Form.ts

```diff
@@ -0,0 +1,87 @@
+import { styled } from '../stitches.config';
```

Comment:
you can add the export of these components in `sigle/src/ui/index.ts` so they can be imported like normal UI components :)